packages/cli/src/utils/input.ts

```diff
@@ -261,8 +274,10 @@ export function getPlaceholder(type: string): string {
     return '0x1df62f291b2e969fb0849d99d9ce41e2f137006e';
   } else if (type === 'string') {
     return 'Hello world';
+  } else if (type === 'tuple' && components) {
```

Comment:
Are there cases when `components` is undefined?

Reply:
There shouldn't if type is tuple, but given I found no docs or guarantees about it, I preferred to err on the safe side.